### DISCUSSION TITLE
How to test my soketi server under load?

### DISCUSSION THREAD

**Discussion by ligonsk** [soketi/pws#430] [Q&A]
I have my Soketi server locally (LAN) on a virtual machine , and a second virtual machine with Laravel project that sends messages to the frontend.

The messages are received so it's working nicely. But I want to try to stress the Soketi server as seen in the repository description:

https://levelup.gitconnected.com/serving-100k-requests-second-from-a-fanless-raspberry-pi-4-over-ethernet-fdd2c2e05a1e?gi=6c822c85ee4f

How can I do that?

**Comment by nathanatimu** [comment]
this, it would be really nice to have a description how to stress test an environment.

jmeter has a plugin for websockets. you can find it [here](https://bitbucket.org/pjtr/jmeter-websocket-samplers/downloads/)
however not sure how compleet it is. it also seems to be missing some sort of authentication that soketi has.. (or i just haven't found it yet)
